app/subscriber/src/features/utils/determinePreview.ts

```diff
@@ -0,0 +1,17 @@
+import { trimWords } from 'features/search-page/utils';
+import { ContentTypeName, IContentModel } from 'tno-core';
+
+export const determinePreview = (item: IContentModel) => {
+  switch (item.contentType) {
```

Comment:
Still not handling transcription correctly.  We can only display the A/V body (which is the transcription) if it has been `content.isApproved`.